src/main/kotlin/constant/Constant.kt

```diff
@@ -0,0 +1,5 @@
+package domain.constant
+
+object Constant {
+    val BOARD_SIZE = 15
```

Comment:
```suggestion
    const val BOARD_SIZE = 15
```
상수 변수라면 const를 붙여보면 어떨까요?